marketing/signals.py

```diff
@@ -21,27 +21,22 @@
 """
 from django.dispatch import receiver
 
-from instance.signals import appserver_spawned
+from registration.signals import betapplication_accepted
 from marketing.models import Subscriber
 
 
-@receiver(appserver_spawned)
+@receiver(betapplication_accepted)
 def register_subscriber(sender, **kwargs):
     """
     Registers a marketing email subscriber for Beta testers
     on first successful appserver spawn. Only registers if user is not
     already registered as subscriber.
     """
-    instance = kwargs['instance']
-    appserver = kwargs['appserver']
-    application = instance.betatestapplication_set.first()  # There should only be one
+
+    application = kwargs['application']
 
     # Ignore if the appserver was not spawned for a beta tester.
     if not application:
         return
```

Comment:
This signal is only triggered if there's a betatest application, so we don't need to re-check this here.
Ref: https://github.com/open-craft/opencraft/pull/850/files#diff-6137d8550b33801d66d4ffcb337e4d2bc3bdb9f1fe2c48911f8589fe277517c5L72
```suggestion
```